falkordb_bulk_loader/relation_type.py

```
@@ -55,65 +55,68 @@ def post_process_header_with_schema(self, header):
-                    self.query_buffer.reltypes.append(self.to_binary())
+                        if self.config.skip_invalid_edges is False:
+                            raise
+                        continue
+                    fmt = "=QQ"  # 8-byte unsigned ints for src and dest
+                    try:
+                        row_binary = struct.pack(fmt, src, dest) + self.pack_props(row)
+                    except SchemaError as e:
+                        raise SchemaError(
+                            "%s:%d %s"
+                            % (self.infile.name, self.reader.line_num, str(e))
+                        )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)